source/agora/network/Client.d

```diff
@@ -168,7 +171,9 @@ public class NetworkClient
         this.exception = new Exception(
             format("Request failure to %s after %s attempts", address,
                 max_retries));
-        this.gossip_timer = this.taskman.setTimer(10.msecs, &this.gossipTask, Periodic.Yes);
+        // Create and stop timer immediately
+        this.gossip_timer = this.taskman.setTimer(GossipDelay, &this.gossipTask, Periodic.No);
+        this.gossip_timer.stop();
```

Comment:
https://github.com/vibe-d/vibe-core/blob/9ca874dc75020b1488a0ac64a1cc5f6471ba15f4/source/vibe/core/core.d#L1243-L1255

But okay, this works :D